nixos/modules/services/networking/iwd.nix

```diff
@@ -15,6 +36,8 @@ in {
       '';
     }];
 
+    environment.etc."iwd/main.conf".source = (pkgs.formats.ini {}).generate "main.conf" cfg.settings;
```

Comment:
Do we want to restart the service whenever this file changes, or leave it to the user to manually restart the service when convenient?

Reply:
I would want to restart whenever the file changes, so I added a `restartTriggers` entry.